content/10.diagnostics.md

```diff
@@ -81,11 +104,22 @@ The authors are now working to get the assay into clinical use [@url:https://www
 
 Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
 Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
+Due to the long incubation times and delayed immune responses of infected patients, serological tests are too insensitive for a diagnosis in the early stages of an infection.
```

Comment:
```suggestion
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
```
I have never heard "insensitive" used for reduced sensitivity, but that doesn't mean I'm right! Please feel free to correct if it is the correct phrasing.

Reply:
No, you're right.